.github/workflows/main.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+jobs:
+  build:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./tests/CI/run-unit-test
```

Comment:
I also wonder if everything should be driven from a single workflow files or from multiples. As we add more things to be run, it might became harder to read the output and it might take a longer time to execute. 

That's probably something to revisit latter. Not in this PR.